src/backend/database_models/seeders/deplyments_models_seed.py

```diff
@@ -112,86 +105,63 @@ def deployments_models_seed(op):
     """
     session = Session(op.get_bind())
 
-    default_user = session.query(User).filter_by(id="user-id").first()
     default_organization = Organization(
         name="Default Organization",
         id="default",
     )
     session.add(default_organization)
     session.commit()
-    default_organization.users.append(default_user)
-    session.commit()
-
-    default_agent = Agent(
-        id="default",
-        version=1,
-        name="Command R+",
-        description="Default agent",
-        preamble="",
-        temperature=0.3,
-        tools=[
-            "web_search",
-            "search_file",
-            "read_document",
-            "toolkit_python_interpreter",
-            "toolkit_calculator",
-            "wikipedia",
-            "google_drive",
-            "arxiv",
-            "example_connector",
-            "pub_med",
-            "file_reader_llamaindex",
-            "wolfram_alpha",
-            "clinical_trials",
-        ],
-        user_id=default_user.id,
-        organization_id=default_organization.id,
-    )
-    session.add(default_agent)
-    session.commit()
 
     # Seed deployments and models
     for deployment in MODELS_NAME_MAPPING.keys():
```

Comment:
I don't think we want to seed with any data?

Reply:
hmm why not?

removing the default agent w/ user-id fixes the migration issues, and also removed the default agent as discussed. we should still seed the model configs, no?

Reply:
I believe we need to keep the models and deployments in the database. I removed the default agent as we discussed, but we still need the models and deployments. I also refactored the seeder to use raw SQL to avoid conflicts with SQLAlchemy model changes.